noise_model_name: GaussianNoiseModel
std_deviations:
    position: [0.8, 0.8, 0.8]
    orientation: [0.1, 0.1, 0.1]
type_noise:
    allowed_semantic_tags:
        - 10
        - 1
        - 14
        - 17
        - 4
        - 5
        - 6
        - 7
        - 8
        - 13
        - 22
        - 18
        - 12
        - 9
        - 11
        - 21
